dist/webgazer.amd.js

```diff
@@ -0,0 +1,6112 @@
hapeToKeepDim",(function(){return X})),n.d(f,"assertAxesAreInnerMostDims",(function(){return Y})),n.d(f,"getAxesPermutation",(function(){return Q})),n.d(f,"getUndoAxesPermutation",(function(){return Z})),n.d(f,"getInnerMostAxes",(function(){return J})),n.d(f,"getBroadcastDims",(function(){return y})),n.d(f,"getReductionAxes",(function(){return x})),n.d(f,"assertAndGetBroadcastShape",(function(){return v})),n.d(f,"assertParamsConsistent",(function(){return A})),n.d(f,"computeOutShape",(function(){return k})),n.d(f,"computePool2DInfo",(function(){return He})),n.d(f,"computePool3DInfo",(function(){return Ke})),n.d(f,"computeConv2DInfo",(function(){return Xe})),n.d(f,"computeConv3DInfo",(function(){return Ye})),n.d(f,"computeDefaultPad",(function(){return Qe})),n.d(f,"tupleValuesAreOne",(function(){return nt})),n.d(f,"eitherStridesOrDilationsAreOne",(function(){return rt})),n.d(f,"convertConv2DDataFormat",(function(){return at})),n.d(f,"PARALLELIZE_THRESHOLD",(function(){return co})),n.d(f,"computeOptimalWindowSize",(function(){return lo})),n.d(f,"nonMaxSuppressionV3",(function(){return Es})),n.d(f,"nonMaxSuppressionV5",(function(){return $s})),n.d(f,"upcastType",(function(){return Js.b})),n.d(f,"getReshaped",(function(){return ei})),n.d(f,"getPermuted",(function(){return ti})),n.d(f,"getReshapedPermuted",(function(){return ni})),n.d(f,"getSliceBeginCoords",(function(){return ri})),n.d(f,"getSliceSize",(function(){return ai})),n.d(f,"prepareAndValidate",(function(){return zr})),n.d(f,"validateUpdateShape",(function(){return Wr})),n.d(f,"validateInput",(function(){return qr})),n.d(f,"calculateShapes",(function(){return Gr})),n.d(f,"SELU_SCALEALPHA",(function(){return fn})),n.d(f,"SELU_SCALE",(function(){return mn})),n.d(f,"shouldFuse",(function(){return Ps})),n.d(f,"ERF_P",(function(){return oi})),n.d(f,"ERF_A1",(function(){return si})),n.d(f,"ERF_A2",(function(){return ii})),n.d(f,"ERF_A3",(function(){return ui})),n.d(f,"ERF_A4",(function(){return ci})),n.d(f,"ERF_A5",(function(){return li})),n.d(f,"warn",(function(){return di})),n.d(f,"log",(function(){return hi})),n.d(f,"mergeRealAndImagArrays",(function(){return pi})),n.d(f,"splitRealAndImagArrays",(function(){return fi})),n.d(f,"complexWithEvenIndex",(function(){return mi})),n.d(f,"complexWithOddIndex",(function(){return gi})),n.d(f,"getComplexWithIndex",(function(){return bi})),n.d(f,"assignToTypedArray",(function(){return yi})),n.d(f,"exponents",(function(){return xi})),n.d(f,"exponent",(function(){return vi})),n.d(f,"segment_util",(function(){return s})),n.d(f,"castTensor",(function(){return wi})),n.d(f,"reshapeTensor",(function(){return Oi})),n.d(f,"linspaceImpl",(function(){return Ci}));var m={};n.r(m),n.d(m,"nonMaxSuppressionV3",(function(){return Es})),n.d(m,"nonMaxSuppressionV5",(function(){return $s})),n.d(m,"split",(function(){return $i})),n.d(m,"tile",(function(){return Ni})),n.d(m,"topkImpl",(function(){return Ri})),n.d(m,"whereImpl",(function(){return tn}));var g=n(5),b=(n(49),n(6));
```

Comment:
We decided not to include precompiled minimized code in the Github Repo as it creates code bloat. This should probably be added to the gitignore.

Reply:
Examples available in www/ includes webgazer.var.js generated file. Ive included these files into repo just because of if someone wants to try out examples after cloning the repo, would not have to install any dependency and build the project